doc-content/enterprise-only/openshift/template-deploy-rhsso-ldap-proc.adoc

```diff
@@ -0,0 +1,106 @@
+[id='template-deploy-rhsso-ldap-{context}-proc']
+= Setting parameters for RH-SSO or LDAP authentication
+
+// local variables depending on the assembly context
+:sso_singleclient!:
+:sso_central!:
+:sso_kieserver!:
+// "typical" is a single KIE server where paramenert are KIE_SERVER_* not KIE_SERVERn_*
+:sso_kieserver_typical!:
+
+ifeval::["{context}"=="freeform-monitor"]
+:sso_central:
+:sso_kieserver:
+endif::[]
+
+ifeval::["{context}"=="freeform-server-managed"]
+:sso_singleclient:
+:sso_kieserver:
+:sso_kieserver_typical:
+endif::[]
+
+ifeval::["{context}"=="freeform-server-immutable"]
+:sso_singleclient:
+:sso_kieserver:
+:sso_kieserver_typical:
+endif::[]
+
+
+If you want to use RH-SSO or LDAP authentication, complete the following additional configuration. Do not configure LDAP authentication and RH-SSO authentication in the same deployment.
```

Comment:
Looks like a prereq to me.

Reply:
Not sure what sounds like a prereq. "Do not configure LDAP and RH-SSO" is not a prereq because you configure them in this procedure and not before. But added the prereq thing to new JIRA.